### PR TITLE
Bug 2061496: Adding message to ControllerStarted:RecentBackup condition

### PR DESCRIPTION
**Pull request by dusk125**
https://bugzilla.redhat.com/show_bug.cgi?id=2061496

### CODE REVIEW

**Comment by dusk125** [comment]
/retitle Bug 2061496: Adding message to ControllerStarted:RecentBackup condition

**Comment by dusk125** [comment]
/retitle Bug 2061496: Adding message to ControllerStarted:RecentBackup condition

File: pkg/operator/upgradebackupcontroller/upgradebackupcontroller.go [hunk 1]
```diff
@@ -103,9 +103,10 @@ func (c *UpgradeBackupController) sync(ctx context.Context, syncCtx factory.Sync
 	clusterOperatorObj := originalClusterOperatorObj.DeepCopy()
 	if !isBackupConditionExist(originalClusterOperatorObj.Status.Conditions) {
 		configv1helpers.SetStatusCondition(&clusterOperatorObj.Status.Conditions, configv1.ClusterOperatorStatusCondition{
-			Type:   "RecentBackup",
-			Status: configv1.ConditionUnknown,
-			Reason: "ControllerStarted",
+			Type:    "RecentBackup",
+			Status:  configv1.ConditionUnknown,
+			Reason:  "ControllerStarted",
+			Message: "Due to a minor version bump, etcd will be taking a snapshot shortly",
```
Comment (tjungblu) [inline pkg/operator/upgradebackupcontroller/upgradebackupcontroller.go:109]: that's quite nit-picky and I'm not super familiar with this controller yet, but how do you know it's a minor version bump?

**Comment by tjungblu** [comment]
@dusk125 the bug targets 4.8, we merge into 4.11 right now - are you going to cherry-pick this all the way down into 4.8? 
We'd need the BZ chain from 4.8->4.11 (I'm sure you're familiar with this after the rebases) in that case for sure.

**Pull request merged**